### PR TITLE
Rewrite and simplify the documentation to make it use-case oriented

### PR DESCRIPTION
**Pull request by mnapoli**
Rewrite and simplify the documentation to make it use-case oriented, and consistent between SQS, SNS and EventBridge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,76 +25,40 @@ return [
...
 ```
 
-Now, let's create our Lambda handler (for example `bin/consumer.php`):
-
-```php
-<?php declare(strict_types=1);
+That's it, messages will be dispatched to that queue.
 
-require dirname(__DIR__) . '/config/bootstrap.php';
+> Note: when running Symfony on AWS Lambda, it is not necessary to configure credentials. The AWS client will read [`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) automatically.
```
Comment (Nyholm) [inline README.md:57]: I am not sure this should be changed or not. However, this is not correct. 

I am pretty sure `AWS_SECRET_ACCESS_KEY` is not available (Or i missed it when I looked).  However `AWS_SESSION_TOKEN` is available and will be picked up automatically.

File: README.md [hunk 2]
```diff
@@ -105,12 +69,43 @@ functions:
...
+return $kernel->getContainer()->get(SqsConsumer::class);
+```
+
+3. Register and configure the `SqsConsumer` service:
+
+```yaml
+# config/services.yaml
+services:
+    Bref\Symfony\Messenger\Service\Sqs\SqsConsumer:
+        public: true
```
Comment (Nyholm) [inline README.md:101]: Usually "autowrite: true" is default in this file. But you might want to add it here. 
```suggestion
        public: true
        autowire: true
```

Comment (mnapoli) [reply README.md:101]: Oh yes good point! I'll add it to the other consumers too.

File: README.md (hunk 1 above)
Comment (mnapoli) [reply README.md]: It actually reads the 3 variables, but that's a detail. I've reworded the sentence to simplify it and avoid the details.

**Pull request merged**